apps/web/src/components/modal-with-close-button.tsx

```diff
@@ -90,20 +92,25 @@ export function ModalWithCloseButton({
           <Dialog.Content
             className={cn('serlo-modal', className)}
             data-modal-state={isOpen ? 'open' : 'closed'}
+            aria-describedby={undefined}
           >
-            {title ? (
-              <Dialog.Title className={cn('serlo-h2', extraTitleClassName)}>
-                {title}
-              </Dialog.Title>
-            ) : null}
+            <Dialog.Title
+              className={cn(
+                'serlo-h2',
+                extraTitleClassName,
+                title ? '' : 'sr-only'
+              )}
+            >
+              {title || altText}
```

Comment:
Why have you added `altText` like this in the modal? For me I would rather have a call like

```typescript
<ModalWithCloseButton text={title ?? altText}/>
```